doc_src/language.rst

```diff
@@ -266,8 +266,8 @@ Consider this helper function::
 Now let's see a few cases::
 
   # Redirect both stderr and stdout to less
-  # (can also be spelt as `&|`)
   print 2>&1 | less
+  print &| less       # equivalent using &|
```

Comment:
this is saying `&|` is equivalent to `&|`, which seems redundant?

I would probably do it like this:

```fish
print 2>&1 | less
# or
print &| less
```